runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -4637,11 +4637,13 @@ void memoryDisclaimLogic(TR::CompilationInfo *compInfo, uint64_t crtElapsedTime,
          }
       }
 
-   // Use logic similar to Data caches above for now
    if (TR::CodeCacheManager::instance()->isDisclaimEnabled())
       {
+      int64_t lateSCCDisclaimTime = compInfo->getPersistentInfo()->getLateSCCDisclaimTime();
+
       // Ensure we don't do it too often
-      if (crtElapsedTime > lastCodeCacheDisclaimTime + TR::Options::_minTimeBetweenMemoryDisclaims)
+      if (lateSCCDisclaimTime == 0 &&
```

Comment:
Oh, I thought we used this option to do a disclaim once after the time specified by this option. Sometimes it's good to do that for debugging so I implemented so that we either disclaim periodically (if the option is 0) or we disclaim once. But let me remove this code to avoid any confusion.

Reply:
Done